### ISSUE TITLE
Problem statement

### ISSUE THREAD

**Issue by ReshmaArumugam** [IBM-EPBL/IBM-Project-34619-1660240015#3]
_The major problems that the farmers of our country are currently facing include Crop Failure, Lack of adequate knowledge, Crop damage due to ignorance/carelessness, Lack of professional assistance, Inaccessibility to agro-tech solutions. The website that we are going to develop will help the farmers to deal with these problems by providing the following aids: Fertilizer suggestion system, Crop Disease Detection System. We will develop a website that will detect crop diseases by scanning the leaves of the crops and recommending the fertilizer that has to be used to get rid of the disease._
